src/dir.props

```diff
@@ -9,11 +9,18 @@
 
   <PropertyGroup>
     <LibPrefix Condition="'$(OSGroup)' != 'Windows_NT'">lib</LibPrefix>
+    <LibSuffix>.so</LibSuffix> 
     <LibSuffix Condition="'$(OSGroup)' == 'Windows_NT'">.dll</LibSuffix>
     <LibSuffix Condition="'$(OSGroup)' == 'OSX'">.dylib</LibSuffix>
-    <LibSuffix Condition="'$(OSGroup)' != 'Windows_NT' and '$(OSGroup)' != 'OSX'">.so</LibSuffix>  
+    <SymbolFileExtension>.dbg</SymbolFileExtension>
+    <SymbolFileExtension Condition="'$(OSGroup)' == 'OSX'">.dwarf</SymbolFileExtension>
+    <SymbolFileExtension Condition="'$(OSGroup)' == 'Windows_NT'">.pdb</SymbolFileExtension> 
+
+    <CMakeBuildDir>$(IntermediateOutputRootPath)corehost\cmake\</CMakeBuildDir>
```

Comment:
The BaseName properties were already defined here (because HostPolicyBaseName is used both in corehost project and sharedframework.proj), and it made more sense to me to define the paths in the same place in case someone wanted to make an update and wants to easily see what they are updating.  I tend to err towards defining things in the props file just because when they are statically defined, it's much easier to track down the value in an msbuild diag log.